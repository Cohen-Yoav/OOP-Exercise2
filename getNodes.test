# getNodes method Testing file

CreateGraph graph1
ListNodes graph1

CreateNode n1 5
AddNode graph1 n1
ListNodes graph1

CreateNode n2 2
AddNode graph1 n2
ListNodes graph1

CreateNode n3 9
CreateNode n4 4
CreateNode n5 5
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5
ListNodes graph1
