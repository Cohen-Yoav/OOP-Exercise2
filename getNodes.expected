# getNodes method Testing file

created graph graph1
graph1 contains:

created node n1 with cost 5
added node n1 to graph1
graph1 contains: n1

created node n2 with cost 2
added node n2 to graph1
graph1 contains: n1 n2

created node n3 with cost 9
created node n4 with cost 4
created node n5 with cost 5
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1
graph1 contains: n1 n2 n3 n4 n5
